SSO-Auth/Config/configPage.html

```diff
@@ -1,15 +1,272 @@
+            >
+              <span>Delete Provider</span>
+            </button>
+          </form>
+
+          <form id="sso-new-oidc-provider" class="esqConfigurationForm">
+            <div class="verticalSection verticalSection-extrabottompadding">
+              <div class="inputContainer">
+                <label
+                  class="inputLabel inputLabelUnfocused"
+                  for="OidProviderName"
+                  >Name/ID of OID Provider:</label
+                >
+                <input
+                  is="emby-input"
+                  id="OidProviderName"
+                  required=""
+                  type="text"
+                />
+                <div class="fieldDescription">
+                  OID Provider Name to add / Update
+                </div>
+              </div>
+              <div class="inputContainer">
+                <label class="inputLabel inputLabelUnfocused" for="OidEndpoint"
+                  >OID Endpoint:</label
+                >
+                <input
+                  is="emby-input"
+                  id="OidEndpoint"
+                  required=""
+                  type="text"
+                />
+                <div class="fieldDescription">
+                  The OpenID endpoint. Must have a .well-known path available.
+                </div>
+              </div>
+
+              <div class="inputContainer">
+                <label class="inputLabel inputLabelUnfocused" for="OidClientId"
+                  >OpenID client ID:</label
+                >
+                <input
+                  is="emby-input"
+                  id="OidClientId"
+                  required=""
+                  type="text"
+                />
+                <div class="fieldDescription">The OpenID client ID.</div>
+              </div>
+              <div class="inputContainer">
+                <label class="inputLabel inputLabelUnfocused" for="OidSecret"
+                  >OID Secret:</label
+                >
+                <input is="emby-input" id="OidSecret" required="" type="text" />
+                <div class="fieldDescription">
+                  The OpenID Secret. Randomly Generated & Shared.
+                </div>
+              </div>
+
+              <div class="checkboxContainer checkboxContainer-withDescription">
+                <label>
+                  <input id="Enabled" name="Enabled" type="checkbox" />
+                  <span>Enabled</span>
+                </label>
+                <div class="fieldDescription checkboxFieldDescription">
+                  Enable the provider
+                </div>
+              </div>
+
+              <div class="checkboxContainer checkboxContainer-withDescription">
+                <label>
+                  <input
+                    id="EnableAuthorization"
+                    name="EnableAuthorization"
+                    type="checkbox"
+                  />
+                  <span>Enable Authorization by Plugin</span>
```

Comment:
This may be a little vague, but the description helps. Let's think about changing this string too.

Reply:
I think taking the help text from the LDAP plugin will help- same concept, but they've expressed it clearer

Reply:
For now I won't touch it, actually

Reply:
Resolved - WIll revisit help text in further iterations